app/src/controllers/user/sagas.js

```diff
@@ -154,35 +158,77 @@ function* generateApiToken({ payload = {} }) {
         }),
       );
     }
-  } catch (err) {
+    yield put(fetchApiKeysAction());
```

Comment:
I am not sure, that we need to fetch all APIKeys. As I understand correctly we already have has all keys.
We store all keys in our store, and when we create new key, we can just add new key to the our store.